types/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@joystream/types",
-  "version": "3.0.0",
-  "description": "Types for Joystream Substrate Runtime - Nara release",
+  "version": "2.2.0",
```

Comment:
Please revert back to 3.0.0. Hmm didn't we publish 4.0.0 which was originally for CRT.. In that case this will need to be 4.1.0 we will just have to skip 3.x

Reply:
I have noticed that setting "^4.0.0" doesn't cause the CI to break

Reply:
So that would be using the published version to npm, not the version in the monorepo.
Setting the dependency in cli to ^4.0.0 is okay, but the types package in the monorepo should be updated to 4.1.0